editor/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./App.css";
+import Editor from "./editor";
+
+function App() {
+  return (
+    <div className="App">
+      <header className="head">
+        <h1 className="heading">My CSS Editor</h1>
+      </header>
+      <main>
+        <Editor />
+      </main>
+      <p className="instructions">
```

Comment:
Would be more obvious to put instructions at the top (you can make the heading smaller) so that they are above the fold on all screen sizes/orientations.